easybuild/framework/easyblock.py

```diff
@@ -2118,20 +2118,33 @@ def update_exts_progress_bar_helper(running_exts, progress_size):
                 # if some of the required dependencies are not installed yet, requeue this extension
                 elif pending_deps:
 
-                    # make sure all required dependencies are actually going to be installed,
-                    # to avoid getting stuck in an infinite loop!
+                    # check whether all required dependency extensions are actually going to be installed;
+                    # if not, we assume that they are be provided by dependencies;
```

Comment:
"we assume that they are be provided" -> "we assume that they are to be provided" (or just "we assume that they are provided")